src/diffusers/loaders/lora_base.py

```diff
@@ -297,6 +306,152 @@ def _best_guess_weight_name(
     return weight_name
 
 
+def _load_lora_into_text_encoder(
```

Comment:
Reason why it's not a part of `LoRABaseMixin` is because not all pipelines support text encoder LoRAs.